test/libsolidity/syntaxTests/constructor/constructible_internal_constructor.sol

```diff
@@ -1,6 +1,8 @@
-contract C {
+abstract contract C {
 	constructor() internal {}
 }
 contract D is C {
-	constructor() public { }
+	constructor() { }
 }
+// ----
+// Warning 2462: (23-48): Visibility for constructor is ignored. If you want the contract to be non-deployable, making it "abstract" is sufficient.
```

Comment:
Is `D` non-deployable because it doesn't explicitly call `C()`? If that's the case, I think the warning message isn't helping much. Maybe message could mention that more explicitly?

Reply:
The warning is on `C` and not on `D`. `D` is deployable (it is not marked abstract).